cli/cmd/portfolio/cmd/seed/generate.go

```diff
@@ -22,6 +22,6 @@ var generateCmd = &cobra.Command{
 func init() {
 	// Define flags for the command.
 	flag.AddMnemonicFlag(generateCmd)
-
+	flag.AddSeedFlag(generateCmd)
 	Command.AddCommand(generateCmd)
```

Comment:
Cosmetic changes: keep empty line above to separate logic blocks of flags definitions and command definition.